examples/util/tflite/TFliteModelImporter.js

```diff
@@ -294,6 +294,7 @@ class TFliteModelImporter {
           inputs = [inputs[0]];
           inputs.push(this._addScalarInt32(newSize[0]));
           inputs.push(this._addScalarInt32(newSize[1]));
+          inputs.push(this._addScalarInt32(options.alignCorners() ? 1 : 0));
```

Comment:
I don't know exactly if Android NNAPI would ignore this extra parameter. If not, NNAPI will fail on the non-atrous model.

Reply:
Good point! In current implementation, NNAPI should report error. We can fix this by ignore the `align_corner` parameter in browser NNAPI backend. Filed https://github.com/intel/webml-polyfill/issues/468 for this.